test/extensions/filters/http/ext_proc/filter_test.cc

```diff
@@ -2261,6 +2272,90 @@ TEST_F(HttpFilterTest, FailOnInvalidHeaderMutations) {
   filter_->onDestroy();
 }
 
+class HttpFilter2Test : public HttpFilterTest, public Http::HttpConnectionManagerImplMixin {};
+
+// Test proves that when onData(data, end_stream) is called before headers response is returned,
+// ext_proc filter will make the data buffered in ActiveStream buffer without triggering a buffer
```

Comment:
s/will make the data buffered in ActiveStream buffer/will buffer the data in the ActiveStream buffer/

Reply:
Done